py3status/modules/yubikey.py

```diff
@@ -0,0 +1,128 @@
+# -*- coding: utf-8 -*-
+"""
+Show an indicator when YubiKey is waiting for a touch.
+
+Configuration parameters:
+    cache_timeout: How often to detect a pending touch request.
+        (default 1)
+    format: Display format for the module.
+        (default '{state}')
+    state_off: Message when the touch is not needed.
+        (default '')
+    state_on: Message when the YubiKey is waiting for a touch.
+        (default 'Y')
+    u2f_keys_path: Full path to u2f_keys if you want to monitor sudo access.
+        (default '')
```

Comment:
I think `(default None)` would look more nice... and then do `if None: ''` thing in `post_config_hook`. We could add `os.path.expanduser(path)` in `post_config_hook` too.